Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -580,8 +597,21 @@ Renderer::Renderer()
 	}
 
 	glGetIntegerv(GL_MAX_SAMPLES, &g_ogl_config.max_samples);
-	if (g_ogl_config.max_samples < 1 || !g_ogl_config.bSupportsMSAA)
+	if (g_ActiveConfig.iMultisampleMode != MULTISAMPLE_SSAA_4X)
```

Comment:
I think proper handling of SSAA is outside of the scope of this PR.
The whole situation of MSAA and SSAA is a huge mess and has always been.
Like on desktop we can properly support MSAA and SSAA
While OpenGL ES 3.2 we can only support SSAA due to it not supporting all features of GL_ARB_texture_multisample which we need(For stereo rendering anyway?).
So really we need to change thoroughly how we handle them, which requires more work than this simple change. Which right now Android devices can't change settings on the fly anyway.